src/test/java/org/jnode/fs/xfs/XfsFileSystemTypeTest.java

```diff
@@ -31,6 +31,15 @@ public void testSupports() throws Exception
             XfsFileSystemType type = new XfsFileSystemType();
             byte[] buffer = new byte[512];
             assertThat(type.supports(partitionTableEntry, buffer, device), is(true));
+            XfsFileSystem xfs = type.create(device, true);
```

Comment:
This class was intended to be a starting point to give an idea of how unit tests would be done. Now that `XfsFileSystemTest` exists, this class can probably be removed.
We usually don't use `System.out.println` in unit tests, the assert functions will print out values when necessary (a fail condition).

Reply:
the class XfsFileSystemTypeTest was deleted.  Thanks